app/assets/stylesheets/govuk_publishing_components/components/_layout-header.scss

```diff
@@ -54,3 +54,11 @@
   background-color: govuk-colour("yellow");
   color: govuk-colour("black");
 }
+
+.gem-c-layout-header .govuk-header__navigation {
```

Comment:
We shouldn't need this nesting, the navigation item modifier will suffice

Reply:
@alex-ju - updated